### PR TITLE
Trongbq vnpt 20231012

### PR DESCRIPTION
**Pull request by quoctrong1x96**

### CODE REVIEW

**Review by quoctrong1x96 — COMMENTED** [review]
OK

**Review by quoctrong1x96 — COMMENTED** [review]
Khá là OK

**Comment by quoctrong1x96** [comment]
Đồng ý

File: .prettierrc.extension.json
Comment (quoctrong1x96) [inline .prettierrc.extension.json:1]: OK

Comment (quoctrong1x96) [reply .prettierrc.extension.json:1]: OK

**Review by quoctrong1x96 — COMMENTED** [review]
OK

File: app.js
Comment (quoctrong1x96) [inline app.js:1]: OK

File: bin/www
Comment (quoctrong1x96) [inline bin/www:1]: oK

File: helpers/apiResponse.js
Comment (quoctrong1x96) [inline helpers/apiResponse.js:1]: ok

File: package-lock.json
Comment (quoctrong1x96) [inline package-lock.json:1]: ok

File: helpers/apiResponse.js
Comment (quoctrong1x96) [reply helpers/apiResponse.js:1]: ok

File: package.json
Comment (quoctrong1x96) [inline package.json:1]: ok

File: src/app.js
Comment (quoctrong1x96) [inline src/app.js:1]: ok

File: src/assets/message.js
Comment (quoctrong1x96) [inline src/assets/message.js:1]: ok

File: src/controllers/addressController.js
Comment (quoctrong1x96) [inline src/controllers/addressController.js:1]: ok

File: src/controllers/authController.js
Comment (quoctrong1x96) [inline src/controllers/authController.js:1]: ok

File: src/controllers/crudController/crudMethods.js
Comment (quoctrong1x96) [inline src/controllers/crudController/crudMethods.js:1]: ok

File: src/controllers/crudController/index.js
Comment (quoctrong1x96) [inline src/controllers/crudController/index.js:1]: ok

File: src/controllers/locationController.js
Comment (quoctrong1x96) [inline src/controllers/locationController.js:1]: ok

File: src/controllers/productController.js
Comment (quoctrong1x96) [inline src/controllers/productController.js:1]: ok

File: src/controllers/typeController.js
Comment (quoctrong1x96) [inline src/controllers/typeController.js:1]: 0ok

File: src/handlers/apiResponse.js
Comment (quoctrong1x96) [inline src/handlers/apiResponse.js:1]: ok

File: src/handlers/errorHandler.js
Comment (quoctrong1x96) [inline src/handlers/errorHandler.js:1]: ok

File: src/helpers/checkFileExtention.js
Comment (quoctrong1x96) [inline src/helpers/checkFileExtention.js:1]: ok

File: src/helpers/loggerWinston.js
Comment (quoctrong1x96) [inline src/helpers/loggerWinston.js:1]: ok

File: src/helpers/mongooseConfig.js
Comment (quoctrong1x96) [inline src/helpers/mongooseConfig.js:1]: ok

File: src/helpers/multerUpload.js
Comment (quoctrong1x96) [inline src/helpers/multerUpload.js:1]: ok

File: src/helpers/swaggerDefine.js
Comment (quoctrong1x96) [inline src/helpers/swaggerDefine.js:1]: ok

File: src/middlewares/jwt.js
Comment (quoctrong1x96) [inline src/middlewares/jwt.js:1]: ok

File: src/models/adminModel.js
Comment (quoctrong1x96) [inline src/models/adminModel.js:1]: ok

File: src/models/countryModel.js
Comment (quoctrong1x96) [inline src/models/countryModel.js:1]: ok

File: src/models/index.js
Comment (quoctrong1x96) [inline src/models/index.js:1]: ok

File: src/models/locationModel.js
Comment (quoctrong1x96) [inline src/models/locationModel.js:1]: ok

File: src/models/productModel.js
Comment (quoctrong1x96) [inline src/models/productModel.js:1]: ok

File: src/models/roleMode.js
Comment (quoctrong1x96) [inline src/models/roleMode.js:1]: ok

File: src/models/typeModel.js
Comment (quoctrong1x96) [inline src/models/typeModel.js:1]: ok

File: src/models/userModel.js
Comment (quoctrong1x96) [inline src/models/userModel.js:1]: ok

File: src/public/test/country workbook.xls
Comment (quoctrong1x96) [inline src/public/test/country workbook.xls:1]: ok

File: src/public/test/country.xlsx
Comment (quoctrong1x96) [inline src/public/test/country.xlsx:1]: ok

File: src/public/views/index.ejs
Comment (quoctrong1x96) [inline src/public/views/index.ejs:1]: ok

File: src/routes/address.js
Comment (quoctrong1x96) [inline src/routes/address.js:1]: ok

File: src/routes/index.js
Comment (quoctrong1x96) [inline src/routes/index.js:1]: ok

File: src/routes/router.js
Comment (quoctrong1x96) [inline src/routes/router.js:1]: ok

File: src/routes/type.js
Comment (quoctrong1x96) [inline src/routes/type.js:1]: ok

File: src/server.js
Comment (quoctrong1x96) [inline src/server.js:1]: ok

File: src/uploads/excel/4bb1894fe7fc77b8783c38943ef0cc51
Comment (quoctrong1x96) [inline src/uploads/excel/4bb1894fe7fc77b8783c38943ef0cc51:1]: ok

File: src/uploads/excel/Phòng đủ nội thất 3tr6 - Quận 3.txt
Comment (quoctrong1x96) [inline src/uploads/excel/Phòng đủ nội thất 3tr6 - Quận 3.txt:1]: ok

File: src/uploads/excel/a35eea68195fb9637349ec5c5b796c6f
Comment (quoctrong1x96) [inline src/uploads/excel/a35eea68195fb9637349ec5c5b796c6f:1]: okk

File: src/uploads/excel/anh Cuong.jpg
Comment (quoctrong1x96) [inline src/uploads/excel/anh Cuong.jpg:1]: ok

**Review by quoctrong1x96 — COMMENTED** [review]
Finish

**Review by ngochtt15 — APPROVED** [review]
phí của bạn 50k

**Review by ngochtt15 — APPROVED** [review]
ủa gì hoài dẫy

**Comment by quoctrong1x96** [comment]
Tôi rất khổ sở trong lần đầu merger cái request này

**Pull request merged**